test/e2e-tests.sh

```diff
@@ -25,13 +25,148 @@
 
 source $(dirname $0)/../scripts/e2e-tests.sh
 
+# Read metadata.json and get value for key
+# Parameters: $1 - Key for metadata
+function get_meta_value() {
+  # local metadata="${ARTIFACTS}/metadata.json"
+  # cat ${metadata} | grep -oEi "\"${1}\" ?: ?\".*\"" | cut -d "\"" -f 4
+  go run knative.dev/test-infra/test/metahelper --get $@
+}
+
 function knative_setup() {
   start_latest_knative_serving
 }
 
+# Run a prow-cluster-operation tool, installing it first if necessary.
+#             $1..$n - parameters passed to the tool.
+function run_prow_cluster_tool() {
+  go run knative.dev/test-infra/test/prow-cluster-operation $@
+}
+
+# Add function call to trap
+# Parameters: $1 - Function to call
+#             $2...%n - Signals for trap
+function add_trap {
+  local cmd=$1
+  shift 1
+  for trap_signal in "$@"; do
```

Comment:
If you quote, it will be a single entry.

```suggestion
  for trap_signal in $@; do
```